src/main/java/org/jitsi/jigasi/CallContext.java

```diff
@@ -381,4 +381,25 @@ public Object getSource()
     {
         return source;
     }
+
+    /**
+     * Return the meeting url of this context
+     *
+     * @return the meeting url
+     */
+    public String getMeetingUrl()
+    {
+        String url = getBoshURL();
+        String room = getRoomName();
+
+        if(url == null || room == null)
+        {
+            return null;
+        }
+
+        url = url.substring(0, url.indexOf("http-bind?"));
```

Comment:
Ugh. This looks flaky. I think the "?" might not always be present. Can we do "/http-bind" and then add "/"? Can we also add a log message (it would make it easier to debug if something here brakes in the future).